<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hoteladminClass</class>
 <widget class="QMainWindow" name="hoteladminClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hoteladmin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="browser"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Комната</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Номер:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="room"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Этаж:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="floor"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Класс:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="classSelection">
        <item>
         <property name="text">
          <string>Класс I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Класс II</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Класс III</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="seaview">
        <property name="text">
         <string>Имеется вид на море</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Общее кол-во мест:        </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberRoom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Кол-во свободных мест:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberFreeRoom"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Кем занят:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="whoTake">
        <item>
         <property name="text">
          <string>пусто </string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Стоимость в сутки:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cost"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Примечание:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="note"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnFill">
          <property name="text">
           <string>Заполнить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCreate">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hoteladmin.qrc"/>
 </resources>
 <connections/>
</ui>
